src/web/services/domainNamesService.ts

```diff
@@ -26,16 +27,29 @@ export async function GetDomainNames(participantId?: number) {
 export async function UpdateDomainNames(
   domainNames: string[],
   participantId?: number
-): Promise<string[]> {
+): Promise<UpdateDomainNamesResponse> {
   try {
     const result = await axios.post<string[]>(
       `/participants/${participantId ?? 'current'}/domainNames`,
       {
         domainNames,
       }
     );
-    return result.data;
+    return { domains: result.data, isValidDomains: true };
   } catch (e: unknown) {
+    if (
+      e instanceof AxiosError &&
+      e?.response?.status === 400 &&
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+      e?.response?.data.message.includes('Invalid Domain Names')
+    ) {
+      return {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call
+        domains: [e?.response?.data.message.replace('Invalid Domain Names:', '')],
+        isValidDomains: false,
+      };
+    }
```

Comment:
It was requested by admin people that the error still says "invalid domain names" so still need to filter it out but also it is a good check that the error is actually from invalid domains